pyproject.toml

```diff
@@ -4,6 +4,43 @@ version = "5.0.0"
 description = "Mimesis: fake data generator."
 authors = ["Isaak Uchakaev <likid.geimfari@gmail.com>"]
 license = "MIT"
+readme = "README.rst"
+homepage = "https://github.com/lk-geimfari/mimesis"
+repository = "https://github.com/lk-geimfari/mimesis"
+documentation = "https://mimesis.readthedocs.io"
```

Comment:
Please, change it to https://mimesis.name

Reply:
@lk-geimfari done!